### PR TITLE
[RFC][PL-434] Create python lib for logging

### PR DESCRIPTION
**Pull request by raffo0707**
https://uniplaces.atlassian.net/browse/PL-434

### CODE REVIEW

File: unipy_logger/uni_logger.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+
+import time
+import datetime
+import os
+import sys
+import json
```
Comment (hfcorreia) [inline unipy_logger/uni_logger.py:7]: add a nl

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(name='unipy_logger',
+      keywords='log, json, uniplaces',
+      version='0.1.4',
+      description='enforce uniplaces log standards and respect uniplaces logging interface.',
+      url='https://github.com/uniplaces/unipy_logger',
+      author='raffaello de pieri',
```
Comment (jkrnak) [inline setup.py]: capitals in your name

**Review by hfcorreia — APPROVED** [review]

**Review by jkrnak — APPROVED** [review]

File: unipy_logger/uni_logger.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+class UniLogger:
+    __instance = None
+    logger = None
+    context = {}
+    messageLevel = ""
+
+    def __init__(self, stream, level):
+        """ Virtually private constructor. """
+        if UniLogger.__instance != None:
+            raise Exception("Logger is already been instantiated")
```
Comment (dominguezmaria) [inline unipy_logger/uni_logger.py:38]: "Logger is already instantiated" or "Logger has already been instantiated" 🇬🇧

Comment (hfcorreia) [reply unipy_logger/uni_logger.py:38]: https://memegenerator.net/img/instances/500x/52806285/watch-out-we-gotta-grammar-nazi-over-here.jpg

Comment (jkrnak) [reply unipy_logger/uni_logger.py:38]: The Queen would be proud. 🍵  👑

**Pull request merged**